pro/src/pages/Home/Venues/Venue.jsx

```diff
@@ -116,6 +116,7 @@ const Venue = ({
   }, [offererId])
 
   let editVenueLink = `/structures/${offererId}/lieux/${id}?modification`
+  let goReimbursementSectionLink = `/structures/${offererId}/lieux/${id}?modification#remboursement`
```

Comment:
```suggestion
  const editVenueLink = `/structures/${offererId}/lieux/${id}?modification`
  const reimbursementSectionLink = `/structures/${offererId}/lieux/${id}?modification#remboursement`
```
puisque les variables changent pas
et une proposition de changement de nom de variable